### ISSUE TITLE
Periodic Job: Job `tz` option not taken into account

### ISSUE THREAD

**Issue by pcreux** [mperham/sidekiq#4833]
Ruby version: ruby 2.6.6p146 (2020-03-31 revision 67876) [x86_64-darwin19]
Rails version: Rails 6.0.3.5
Sidekiq / Pro / Enterprise version(s): sidekiq (6.1.3) / sidekiq-pro (5.2.1) / sidekiq-ent (2.2.2)

---

Enterprise Web UI displays all periodic jobs to run in the system timezone. It seems to be ignoring `register`'s `tz` option.

<img width="997" alt="Screen Shot 2021-03-04 at 11 00 35 AM" src="https://user-images.githubusercontent.com/45299/110015587-e283ca00-7cd8-11eb-8aae-6b029c74b589.png">

```ruby
# config/initializers/sidekiq.rb
require 'sidekiq-ent/web'

Sidekiq.configure_server do |config|
  # ...
  config.periodic do |cron|
    cron.register "1 0 * * *", "System::DailyHealthCheckJob"
    cron.register "2 0 * * *", "RefreshAllSamlMetadataJob", tz: ActiveSupport::TimeZone.new('UTC')
    cron.register "3 0 * * *", "VacuumTableJob", args: ["public", "organizations"], tz: ActiveSupport::TimeZone.new('EST')
  end
  # ...
end
```

A similar issue happens on `sidekiq-ent (2.2.1)`. I also tried registering classes instead of strings with no success.

**Comment by mperham** [comment]
Unfortunately this is a current limitation in how crons are registered. The Web UI runs in the client context and cron jobs are registered in the server context, so the Web UI does not know the active TZ for each job. I will think about how to solve this.

**Comment by pcreux** [comment]
Ok, so while the time is displayed in the server's timezone it will run in the timezone registered in the initializer. I'm going to stick with UTC then because that's quite confusing. :)

**Comment by OrrYakobi** [comment]
@mperham I see you closed the issue. Was there a solution to this? 

May I suggest using the [in_time_zone](https://apidock.com/rails/DateTime/in_time_zone) method in that view?

Would be happy to implement this myself in a pull request, but I am assuming you don't allow pull requests for the enterprise repo.

**Comment by mperham** [comment]
This was fixed in 2.2.2 I believe. Please correct me if I’m wrong.

**Comment by OrrYakobi** [comment]
I am on 2.2.2 and facing the same issue. 

I also just realized that my proposed solution doesn't exactly work.

**Comment by mperham** [comment]
Ah, it's fixed on master, will be in 2.2.3.

**Comment by OrrYakobi** [comment]
Thank you!
